src/stream/src/from_proto/hop_window.rs

```diff
@@ -83,6 +83,7 @@ impl ExecutorBuilder for HopWindowExecutorBuilder {
         };
         let window_slide = node.get_window_slide()?.into();
         let window_size = node.get_window_size()?.into();
+        let window_offset = node.get_window_offset()?.into();
```

Comment:
Considering the compatibility, we need to give a default offset if this field does not exist in the proto.